scripts/check-pr.sh

```diff
@@ -19,6 +19,27 @@
 # NOTE: must be run from the repository root directory to correctly work!
 # NOTE: no `set -e`, failure of this script should not invalidate the build.
 
+VERBOSE=false
+
+while getopts ":v" opt; do
+  case "$opt" in
+  v)
+    VERBOSE=true
+    ;;
+  *)
+    echo "This argument is not valid for this script."
+    ;;
+  esac
+done
+
+if [[ "$VERBOSE" = true ]]; then
```

Comment:
For consistency with the rest of the script
```suggestion
while getopts ":v" opt; do
  case $opt in
  v)
    VERBOSE=true
    ;;
  *)
    echo "This argument is not valid for this script."
    ;;
  esac
done

if [[ $VERBOSE == true ]]; then
```